runway_model.py

```diff
@@ -19,28 +20,66 @@ def pil_rect_to_x_y_w_h(pil_rect):
     h = pil_rect[3] - y
     return x, y, w, h
 
+USE_CUDA = False
+LAST_LABEL_IMAGE_ARR = None
+LAST_LABEL_ENCODINGS = None
+FACE_MATCH_TOLERANCE = 0.6
+
+def get_model_kwargs():
+    global USE_CUDA
+    if USE_CUDA:
+        return {
+            'model': 'cnn'
+        }
+    else:
+        return {}
+
+@runway.setup(options={ 'match_tolerance': number(min=0.1, max=1.0, step=0.1, default=FACE_MATCH_TOLERANCE) })
```

Comment:
This can be a command input instead of a setup parameter.

Reply:
Ah that's what I was wanting actually, but I figured doing that would create a main UI window input (like text or image), not a slider on the panel. That's great!

Reply:
Yes, we need to document which data types end up on the main input view and which ones end up in the panel!